src/Layer.cpp

```diff
@@ -15,6 +15,12 @@ m_total_offset(j["__pxTotalOffsetX"].get<int>(), j["__pxTotalOffsetY"].get<int>(
 m_opacity(j["__opacity"].get<float>()),
 m_grid_size({j["__cWid"].get<int>(), j["__cHei"].get<int>()})
 {
+    auto& overrideTilesetUid = j["overrideTilesetUid"];
+    if (!overrideTilesetUid.is_null())
+    {
+        m_overrideTileset = &w->getTileset(overrideTilesetUid.get<int>());
+    }
```

Comment:
```suggestion
    if ( !j["overrideTilesetUid"].is_null() ) {
        m_override_tileset = &w->getTileset(j["overrideTilesetUid"].get<int>());
    }
```

We probably don't need to have an intermediate variable here imo

Reply:
I generally prefer the intermediate so that the field string doesn't have to be written (and potentially mispelled) twice.
In this case, I wouldn't expect this key name to change and we're only checking them here, so no concerns.